osm2streets/src/lanes/mod.rs

```diff
@@ -355,3 +356,73 @@ impl fmt::Display for Direction {
         }
     }
 }
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize, Deserialize)]
+pub enum LtrLaneNum {
```

Comment:
Before I read the below comment, I was confused what this meant. Always counting from the left, its the 0th, 1st, 2nd, etc lane that's forwards or backwards, regardless of contraflow. What about `lanes:both_ways`?

Reply:
Pretty much that (but starting at 1). Contraflow lanes are not mentioned anywhere, but a strict reading of "the forward lanes in LtR order" can be followed by mappers and consumers equally well. Does `placement:backward=left_of:3` look like ` V|^^VV`?

Like tagging `turn:lanes[:forward]=left|left;through|through`, three lanes, 1, 2, 3. 

This is where i need to summarise the idea in comments! And link to the wiki quotes.